webapp-django/crashstats/manage/admin.py

```diff
@@ -197,3 +198,23 @@ def graphics_devices_lookup(request):
         return result
     else:
         return http.HttpResponseBadRequest(str(form.errors))
+
+
+@superuser_required
+def debug_view(request):
+    """This view is for ephemeral debugging of issues"""
+
+    context = {}
+
+    # Map of key -> val that will get displayed in a big table in the debug
+    # view in the order they were inserted
+    debug_info = OrderedDict()
+
+    # Add IP address related headers to figure out rate-limiting issues. #1475993
+    for meta_header in ['HTTP_X_FORWARDED_FOR', 'REMOTE_ADDR']:
```

Comment:
Can you also add HTTP_X_REAL_IP? We're currently doing `proxy_set_header X-Real-IP $remote_addr;` in nginx, and having it here will give me some extra wiggle room for debugging since the middleware doesn't touch it.

Reply:
Sure!